File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int currCons = 0;
        int maxCons = 0;
        for(int i=0;i<nums.length;i++){
        if(nums[i]==1){
            currCons++;
        }else{
            currCons=0;
        }
        maxCons=Math.max(currCons,maxCons);
        }
        return maxCons;
    }
    
}

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        curr = 0
        maxi = 0

        for num in nums:
            if num == 1:
                curr += 1
                maxi = max(maxi, curr)
            else:
                curr = 0

        return maxi
        